lib/ipmi_backend_utils.pm

```diff
@@ -145,8 +146,21 @@ sub umount_installation_disk {
     assert_script_run("ls $mount_point");
 }
 
+#Get the partition where the new installed system is installed to
 sub get_installation_partition {
-    my $partition = script_output("fdisk -l | grep \"^\/dev\/sda\.\*\\\*\" | cut -d ' ' -f 1");
+    my $partition = '';
+    if (sle_version_at_least('15')) {
+        my $y2log_file                = '/var/log/YaST2/y2log';
+        my $root_partition_commit_msg = script_output("grep \"Commit Action \\\"Adding mount point / of .* to /etc/fstab\" $y2log_file");
```

Comment:
I am seeing too much quote scaping here. What about:

```perl
my $root_partition_commit_msg = script_output(qq{grep 'Commit Action "Adding mount point / of .* to /etc/fstab' $y2log_file});
```

Reply:
Good way. Will change to it.